packages/services/api/src/modules/schema/providers/schema-publisher.ts

```diff
@@ -600,13 +565,87 @@ export class SchemaPublisher {
             safeSchemaChanges: contract.schemaChanges?.safe ?? null,
           })) ?? null,
       });
+    } else if (checkResult.conclusion === SchemaCheckConclusion.Skip) {
+      if (!comparedVersion || !comparedSchemaVersion) {
+        throw new Error('This cannot happen :)');
```

Comment:
"Hey Siri, remind me about this comment in 2 months from now"

Reply:
Either this or refactor everything and pull the checksum check out of the orchestrator in order to please TypeScript 😇